src/why3prop.ml

```diff
@@ -0,0 +1,135 @@
+(** Translation of lambdapi goals to why3 goals in propositional case *)
+
+open Terms
+open Extra
+
+exception NoGoalTranslation
+
+(* a map from lambdapi terms to Why3 constants *)
+type cnst_table = (term * Why3.Term.lsymbol) list
+
+(* a map from labels to Why3 terms. *)
+type ctxt_labels = Why3.Term.term StrMap.t
```

Comment:
Where is defined the notion of "label"?

The name ctxt_labels is not clear. This suggests that this is a list of labels of some sort but this is not the case. This is a map. A suggestion: term_to_why3symbol_map and string_to_why3term_map.

What is the relation between cnst_table and ctxt_labels?

Reply:
Adapted in e963c2e8677e9bfa6608c2268a47499eb15d49a4.